_extras/checkout.md

```diff
@@ -38,8 +38,9 @@ To get there, try looking for a link that says "Improve this page". In this curr
 
 ![Screen shot of the header for this page with Improve this page circled](../fig/Improve_this_page.svg){: width="800px" }
 
-Alternatively, you can find links to the GitHub repositories for each lesson next to the links to the rendered lessons on the websites listing Data Carpentry,
-Library Carpentry, or Software Carpentry workshops.
+Alternatively, you can find links to the GitHub repositories for each lesson next to the links to the lessons sites on the websites listing [Data Carpentry](https://datacarpentry.org/lessons/), [Library Carpentry](https://librarycarpentry.org/lessons/), or [Software Carpentry](https://software-carpentry.org/lessons/) lessons:
+
+![Screen shot of Sofware Carpentries Lessons list in a browser with Links to Lessons Repositories circled](../fig/Instructor_Training_Repo_0.png){: width="800px" }
```

Comment:
@dpshelio since you have more familiarity with the workbench, do you foresee any issue with fixing the width here?  I was concerned setting it here might break mobile device view? but I do not know the order of how css in different places is processed.

Reply:
I can answer this: no, this will not break the mobile device view in the workbench because [figures in the workbench have a maximum width of 100%](https://github.com/carpentries/varnish/blob/85363d9e15ee65c0ccff914d091c7369c252b2ff/source/stylesheets/styles.css.scss#L182-L184).